src/less/radio/base/radio.less

```diff
@@ -4,7 +4,7 @@
 .radio {
     display: inline-flex;
     font-size: @font-size-16;
-    vertical-align: middle;
+    vertical-align: text-bottom;
```

Comment:
I'm curious about why it needed changing to `text-bottom`...?

Reply:
It wast lining up properly with `middle`. You can try yourself and let me know what you think.